test_scripts/Polices/build_options/026_ATF_P_TC_PTU_PM_Sets_Status_UPDATE_NEEDED_HTTP.lua

```diff
@@ -127,17 +128,18 @@ function Test:RAI_PTU()
         function()
           log("SDL->MOB: N: OnPermissionsChange")
         end)
-      :Times(1)
     end)
+
+  commonTestCases:DelayedExp(65000) -- wait exchange_after_x_seconds(t0) + tollerance 5 sec
 end
 
 Test["Starting waiting cycle [" .. attempts * 5 .. "] sec"] = function() end
 
-for i = 1, attempts do
-  Test["Waiting " .. i * 5 .. " sec"] = function()
-    os.execute("sleep 5")
-  end
-end
+-- for i = 1, attempts do
```

Comment:
@dboltovskyi, if you don't put delay in the test RAI_PTU ATF will wait default timeout of 10 sec, but we need wait of 60 sec for retry sequence. Waiting outside will not to catch the expected notification.